src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -178,7 +221,10 @@ export class EditConcept extends Component {
 
     const regx = /^[a-zA-Z\d-_]+$/;
     if (regx.test(this.state.id) && this.state.datatype && this.state.concept_class) {
-      this.props.updateConcept(this.conceptUrl, this.state, this.props.history);
+      const unRetiredMappings = mappings.filter(m => m.retired === false);
```

Comment:
How is the above related to the ticket?

Reply:
When recreating a reference to a concept, the API defaults all Mappings of that Concept to `retired: false` therefore this line is making sure that the retired mappings are not set to true.